ZuluContent/Mobiles/AI/SpiderAI.cs

```diff
@@ -0,0 +1,51 @@
+//
+// This is a first simple AI
+//
+//
+
+using System;
+using System.Threading.Tasks;
+using Server.Items;
+using Server.Network;
+
+namespace Server.Mobiles
+{
+    public class SpiderAI : MeleeAI
+    {
+        public SpiderAI(BaseCreature m) : base(m)
+        {
+        }
+
+        private async Task SpitWeb(Mobile target)
```

Comment:
Are we going to be adding more functionality to the Spider AI to need an extra AI Type?

You can accomplish the same thing in the `OnThink` method in `BaseCreature`, which is how fire breath for dragons is implemented.

Reply:
ah yeah didn't think about that actually, can probably just implement this in OnThink then and use a HasWeb property